app/views/shared/_request_status_information.html.erb

```diff
@@ -9,7 +9,7 @@
   <% destination_name = LibraryLocation.library_name_by_code(current_request.destination) %>
   <% if destination_name.present? %>
     <dt>
-      <%= label_for_pickup_destinations_dropdown(current_request.pickup_destinations) %>
+      Will be delivered to
```

Comment:
Is this (and the test changes) an intentional change from the current behavior? I think the subtle intent is to indicate whether the patron has any choice about the delivery location, and it seems like we've lost that?

Reply:
This seemed weird that the status display would display two different labels "Will be delivered to" and "Delivered to" depending on whether or not there was a choice.  The function that was used here was intended as a label for a dropdown, but this is not a dropdown.  


Do you want to retain this behavior? I can revert this part, but it just seems wrong.

Reply:
I would err on the side of keeping the current behavior. It was important at the time and I think there's a UX argument to be made about using consistent labels between the form and the email.

If we want to change it, I think it's important to do so transparently and not in a refactoring PR.